blog/2025-03-01-pants-2-25/index.mdx

```diff
@@ -0,0 +1,49 @@
+---
+  authors: [huonw]
+  tags: [announcement]
+  # Date of the actual release
+  date: 2025-02-19
+---
+
+# Pants 2.25.0 is released!
+
+<CaptionedImg src={require("./splash.jpeg").default}>
+  Photo by [Agê Barros](https://unsplash.com/@agebarros) on
+  [Unsplash](https://unsplash.com/photos/a-close-up-of-a-silver-watch-face-rBPOfVqROzY)
+</CaptionedImg>
+
+{/* truncate */}
+
+We are pleased to announce Pants [2.25.0](https://github.com/pantsbuild/pants/blob/2.25.x/docs/notes/2.25.x.md), the latest release of [Pantsbuild, the scalable and ergonomic build system](https://www.pantsbuild.org/). To update, set `pants_version = "2.25.0"` in your `pants.toml`. If you're not using Pants yet, [get started now](https://www.pantsbuild.org/2.25/docs/getting-started).
+
+Thank you to [Klaviyo](https://www.klaviyo.com/) and [Normal Computing](https://normalcomputing.ai/) for their Platinum tier support throughout this release!
```

Comment:
Despite #348, I believe that they were still sponsors for the 2.25 release development cycle, hence worth calling out.